core/src/com/unciv/logic/battle/Battle.kt

```diff
@@ -258,7 +258,10 @@ object Battle {
             return
 
         var XPModifier = 1f
+        // keep until deprecated
```

Comment:
Please add what version this was added into so we'll know how much time has passed :)